frontend/src/components/Shared/SearchBar/SearchBar.jsx

```
@@ -1,43 +1,462 @@
+          
+          <input
+            type="search"
+            value={searchQuery}
+            onChange={(e) => setSearchQuery(e.target.value)}
+            placeholder={placeholder}
+            className="w-full py-2 pl-10 pr-10 bg-gray-800/70 border border-gray-700 focus:border-blue-500 rounded-full text-white placeholder-gray-400 outline-none transition-all duration-300 focus:ring-2 focus:ring-blue-500/50 shadow-lg"
+            onFocus={() => {
+              setIsSearchFocused(true);
+              setShowSuggestions(true);
+            }}
+            autoFocus={autoFocus}
```

Biome (1.9.4)

[error] Avoid the autoFocus attribute.

Unsafe fix: Remove the autoFocus attribute.


(lint/a11y/noAutofocus)